src/AuthServer.ts

```diff
@@ -38,7 +38,7 @@ export class AuthServer {
   };
 
   private httpListener = async (): Promise<void> => {
-    const requestState = Math.random().toString(16).substr(2, 20);
+    const requestState = Math.random().toString(16).substring(2, 20);
```

Comment:
AFAIK, this is not the same right?

`.substr(2, 20`) returns 20 characters starting from index 2.
`.substring(2, 20)` returns characters between index 2 and index 19 (a total of 18 characters).

Reply:
I think that in this it doesn't matter how many characters are returned.

I'm trying to check the output from Math.random().toString(16) and it always returns 13 characters after decimal separator. Maybe using `substring` is not necessary.